app/models/gdrive/sync_listener.rb

```diff
@@ -63,7 +63,7 @@ def groups_membership_committed(membership)
 
     def groups_affiliation_committed(affiliation)
       group = affiliation.group
-      return unless group.everybody?
+      return unless group.present? && group.everybody?
```

Comment:
Need to check that group is present first so an exception is not raised when doing a cluster.destroy